frontend/src/stores/EntityCountsStore.ts

```diff
@@ -0,0 +1,20 @@
+import SimpleStore from './SimpleStore';
+import { CountQueryResult } from '../shared/queries';
+
+type EntityCounts = { nodes: number; edges: number };
+
+/**
+ * Contains the current state of the number of entities to be displayed.
+ */
+export default class EntityCountsStore extends SimpleStore<EntityCounts> {
```

Comment:
As there is now a state-store for the limit-state, The limit can directly consumed by the filter query store. Currently, this store is directly manipulated in the max-entities slider.

Reply:
the `EntityCountsStore` should contain the current number of the entity slider and not the limit. I think the limit does not need to be stored since it is only manipulated in the initialization.

Reply:
or do you mean, by "limit" the current number? If you mean it I think it is a bit confusing.

Reply:
Limit is correct. The limit is the number that limits the amount of entities, so is the currently value of the slider and is used as an argument in the query. See https://github.com/amosproj/amos-ss2021-project2-context-map/blob/56b5f8bb5ad7a0c3f7c07ae6a81630ba0757f918/shared/src/queries/QueryBase.ts#L5-L22